pymc/gp/cov.py

```diff
@@ -198,19 +258,73 @@ def merge_factors(self, X, Xs=None, diag=False):
                     factor_list.append(at.diag(factor))
                 else:
                     factor_list.append(factor)
+
+            else:
+                factor_list.append(factor)
+
+        return factor_list
+
+    def _merge_factors_psd(self, omega):
+        """Called to evaluatate spectral densities of combination kernels when possible.  Implements
+        a more restricted set of rules than `_merge_factors_cov` -- just additivity of stationary
```

Comment:
```suggestion
        """Called to evaluatate spectral densities of combination kernels when possible.

        Implements
        a more restricted set of rules than `_merge_factors_cov` -- just additivity of stationary
```

Numpydoc style says that docstrings should start with one sentence, followed by an empty line IIRC.
The docs build warns about these things..